doc/kb_sdk_impl_methods.md

```diff
@@ -72,8 +72,8 @@ read from the scratch folder.
 
 ```python
 import os
-from AssemblyUtil.AssemblyUtilClient import AssemblyUtil
-from KBaseReport.KBaseReportClient import KBaseReport
+from Assemblyutil.AssemblyutilClient import AssemblyUtil
```

Comment:
This doesn't match what I see.  Are you sure?

Reply:
Same for the next too.

Reply:
@nixonpjoshua can you reply to @scanon's questions?